### PR TITLE
adding in overview dashboard and capture loss autoextractor

### PR DESCRIPTION
**Pull request by kris-watts-gravwell**
Adds new dashboard and extractor for corelight_capture_loss

Kit also now depends on the Gravwell kit so we can use the tag aggregates

Screenshot from 2023-05-08 08-40-58

### CODE REVIEW

File: corelight/MANIFEST [hunk 1]
```diff
@@ -981,12 +981,27 @@
 			"Name": "corelight_x509",
 			"Type": 4,
 			"Hash": "0000000000000000000000000000000000000000000000000000000000000000"
+		},
+		{
+			"Name": "4a6a2042-faaf-4d0a-8fa4-df8a90d3aa65",
```
Comment (ashnwade) [inline corelight/MANIFEST:986]: Is this really the name you intended to use?

File: corelight/dashboard/4a6a2042-faaf-4d0a-8fa4-df8a90d3aa65.meta [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+				"query": "tag=_aggs_tags ax tag~corelight | stats sum(entries) by tag | chart sum by tag limit 64",
+				"color": null
+			},
+			{
+				"alias": "Data Rates",
+				"query": "tag=_aggs_tags ax tag~corelight | stats sum(bytes) as data by tag | chart data by tag limit 64",
+				"color": null
+			},
+			{
+				"alias": "Search 3",
```
Comment (ashnwade) [inline corelight/dashboard/4a6a2042-faaf-4d0a-8fa4-df8a90d3aa65.meta:25]: "alias": "Overview" ?

**Review by ashnwade — COMMENTED** [review]
Just double checking some naming, but other than that, LGTM

File: corelight/MANIFEST (hunk 1 above)
Comment (kris-watts-gravwell) [reply corelight/MANIFEST:986]: yes, the Name is just the UUID of the dashboard.

File: corelight/dashboard/4a6a2042-faaf-4d0a-8fa4-df8a90d3aa65.meta (hunk 2 above)
Comment (kris-watts-gravwell) [reply corelight/dashboard/4a6a2042-faaf-4d0a-8fa4-df8a90d3aa65.meta]: agreed, fixed

**Comment by kris-watts-gravwell** [comment]
PTAL

**Review by ashnwade — APPROVED** [review]

**Pull request merged**